lib/parsers/yarn-lock-parse.ts

```diff
@@ -59,8 +20,6 @@ export class YarnLockParser implements LockfileParser {
       );
     }
     this.yarnLockfileParser = require('@yarnpkg/lockfile');
```

Comment:
not a fan of this, can we just import and use the library direct?

Reply:
If you don't support node 6 then this can indeed moved to top level

Reply:
@dkontorovskyy can you please fix it in your branch? I don't have access to it

Reply:
@regevbr sure :) will take a look later today :)